.test-infra/jenkins/job_PostCommit_Java_Tpcds_Spark.groovy

```diff
@@ -32,73 +35,42 @@ NoPhraseTriggeringPostCommitBuilder.postCommitJob('beam_PostCommit_Java_Tpcds_Sp
 
       // Gradle goals for this job.
       steps {
-        shell('echo "*** RUN TPCDS IN BATCH MODE USING SPARK 2 RUNNER ***"')
```

Comment:
Looks like this file contains a copy-paste error... you're running the tests for Spark 3 twice, but not the ones for Spark 2